### ISSUE TITLE
Refresh - Aufstellung Rechnungen u.a.

### ISSUE THREAD

**Issue by Mucki526** [jlawyerorg/j-lawyer-org#2217]
Bei der Aufstellung Rechnungen ist es mir wieder aufgefallen. Wenn man in das Menue reingeht, muss man immer einen manuellen refresh machen, bevor man sich auf die Liste verlassen kann. Ist auch an anderen Stellen so, oder auch im Thunderbirdplugin für die Aktenliste. Oder man muss erst aus einem Menue raus- und wieder reingehen, dass es aktualisiert ist. 

Ich erinnere mich, dass die Linux-Communitiy insoweit bei ihren editoren früher insoweit immer bzgl. des manuellen refresh von "feature - kein Bug" sprach, aber für die tägliche Arbeit als Anwalt ist das in meinen Augen einfach nur eine mögliche Fehlerquelle. 

Ich würde mir hier an dieser Stelle für die Rechnungsauflistung wünschen, dass einfach beim Aufruf einfach automatisch ein Refresh gemacht wird. 

**NOTE: Please search existing tickets to avoid duplicates **
**HINWEIS: Bitte vorab die Ticketsuche verwenden, um Doppeleinträge zu vermeiden **

**Is your feature request related to a problem? Please describe. / Welches Problem soll durch die Erweiterung / Verbesserung gelöst werden?**
Description / Beschreibung

**Describe the solution you'd like / Beschreibe die gewünschte Lösung**
Description / Beschreibung

**Additional context / Sonstiges **
Sonstige Informationen / Screenshots / ...

**Comment by Mucki526** [comment]
oh.... doch, es taucht auf, irgendwo in den Neunziger Rechnungsnummern (sorry, das habe ich übersehen, aber es sortiert sich offensichtlich nicht beim aktualisieren)
![grafik](https://github.com/jlawyerorg/j-lawyer-org/assets/37142896/355aa153-3f77-4eae-a1ad-6b8cf50caf58)

**Comment by j-dimension** [comment]
@Mucki526 Gehören diese Kommentare vielleicht eher zu #2216 ?

**Comment by Mucki526** [comment]
Ja, sorry, mein Fehler, tut mir leid... beim klicken nicht aufgepasst
